src/order-history/reducers.js

```diff
@@ -0,0 +1,40 @@
+import { FETCH_ORDERS } from './actions';
+
+export const initialState = {
+  loadingOrders: false,
```

Comment:
Stylistically, we could probably just name these `loading` and `error`... They're already part of the orderHistory reducer, and it's the only thing we need to load.  

(Also, I've been adding `loaded` as well, as it's proved useful here and there.)